src/Components/TaskForm.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import {
+    Box,
+    Button,
+    InputLabel,
+    TextField,
+    Typography
+} from '@material-ui/core';
+import { FormikHelpers, useFormik } from 'formik';
+import schema from './Validation/schema';
+import { ITask, PropsTaskForm } from '../interfaces/task';
+
+const TaskForm: FC<PropsTaskForm> = ({ onTaskAdded }: PropsTaskForm) => {
+    const handleSubmitAddTask = (values: ITask, { resetForm }: FormikHelpers<ITask>) => {
```

Comment:
I suggest puting this inside formik hook.

Reply:
Resolved on 1ed844153e5a8d882aec402fb661ac5e227ae84e